File: test/fixtures/output/php/guzzle/application-form-encoded.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'form_params' => [
    'foo' => 'bar',
    'hello' => 'world'
  ],
  'headers' => [
    'content-type' => 'application/x-www-form-urlencoded',
  ],
]);

echo $response->getBody();

File: test/fixtures/output/python/python3/application-form-encoded.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "foo=bar&hello=world"

headers = { 'content-type': "application/x-www-form-urlencoded" }

conn.request("POST", "/anything", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
